lib/dialects/oracle/schema/oracle-tablecompiler.js

```diff
@@ -86,14 +87,20 @@ class TableCompiler_Oracle extends TableCompiler {
     );
   }
 
-  primary(columns, constraintName) {
+  primary(columns, constraintName, deferrable) {
```

Comment:
Generally we try to avoid adding many positional parameters, as these become very confusing very fast.
Could you add options object as a third parameter and put `deferrable` there?

Reply:
Are you suggesting having it like this? ```primary(columns,consntraintName,options={[deferrable:string]}```
Or should it be ```primary(columns,constraintName|options={[constrantName:string],[deferrable:string]})```
This way we can still make sure that code using older version still work and allow user to provide ```deferrable``` without having to specify ```constraintName```
Then maybe in the future we can switch to have only options```primary(columns,options={[constrantName:string],[deferrable:string]})```

Reply:
Yes, it is important to maintain backwards compatibility with the existing code. Second approach seems cleaner, since it doesn't make sense to require constraintName to be specified in order to be able to use options